ui/scss/src/_common.scss

```diff
@@ -105,3 +105,39 @@
 .icon-externally-managed {
   color: var(--craftercms-medium4);
 }
+
+.notifyjs-material-base {
+  color: #fff;
+  padding: 6px 16px;
+  font-size: 0.875rem;
+  box-shadow: 0px 3px 5px -1px rgba(0, 0, 0, 0.2), 0px 6px 10px 0px rgba(0, 0, 0, 0.14), 0px 1px 18px 0px rgba(0, 0, 0, 0.12);
+  align-items: center;
+  font-weight: 400;
+  line-height: 1.43;
+  border-radius: 4px;
+  background-color: #313131;
+  display: flex;
+  align-items: center
+}
+
+.notifyjs-material-success {
+  color: #fff !important;
+  background-color: #43a047 !important
+}
+
+.notifyjs-material-error {
+  color: #fff !important;
+  background-color: #d32f2f !important
+}
+
+.notifyjs-material-icon-root {
+  margin-inline-end: 8px;
+  fill: currentColor;
+  width: 1em;
+  height: 1em;
+  display: inline-block;
+  font-size: 16px;
+  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
+  flex-shrink: 0;
+  user-select: none
+}
```

Comment:
Looks like you moved the snack styles here but didn't remove them from it's original place? They should be common but not duplicated.